docs/components/base/sensor-controlled.md

```diff
@@ -73,9 +73,10 @@ The following attributes are available for `sensor-controlled` bases:
 <!-- prettier-ignore -->
 | Name | Type | Required? | Description |
 | ---- | ---- | --------- | ----------- |
-| `movement_sensor` | array | **Required** | Array with the `name`s of any movement sensors on your base you want to gather feedback from. The driver will select the first movement sensor providing appropriate feedback for either the `SetVelocity()` or the `Spin()` endpoint. <br> If your sensor has an adjustable frequency or period, set the frequency to something greater than or equal to the default base control loop frequency of 10Hz, or set the period to something less than or equal to the corresponding period of 100msecs. |
+| `movement_sensor` | array | **Required** | Array with the `name`s of any movement sensors on your base you want to gather feedback from. The driver will select the first movement sensor providing appropriate feedback for either the `SetVelocity()` or the `Spin()` endpoint. <br> If your sensor has an adjustable frequency or period, set the frequency to something greater than or equal to the `control_frequency_hz`. A higher frequency will generally result in better performance of the base control loop. |
 | `base` | string | **Required** | String with the `name` of the base you want to wrap with sensor control. |
 | `control_parameters` | object | Optional | A JSON object containing the coefficients for the proportional, integral, and derivative terms for linear and angular velocity. If you want these values to be auto-tuned, you can set all values to 0: `[ { "type": "linear_velocity", "p": 0, "i": 0, "d": 0 }, { "type": "angular_velocity", "p": 0, "i": 0, "d": 0 } ]`, and `viam-server` will auto-tune and log the calculated values. Tuning takes several seconds and spins the motors. Copy the values from the logs and add them to the configuration once tuned for the values to take effect. If you need to auto-tune multiple controlled components that depend on the same hardware, such as a sensor controlled base and one of the motors on the base, run the auto-tuning process one component at a time. For more information see [Feedback control](#feedback-control). |
+| `control_frequency_hz` | float | Optional | Adjusts the frequency that the base control loop runs at. Increasing the frequency will generally result in a better controller, provided the movement sensors can support the higher frequency. The default base control loop frequency is 10Hz. |
```

Comment:
```suggestion
| `control_frequency_hz` | float | Optional | Adjusts the frequency that the base control loop runs at. 
A higher frequency will generally result in more stable behavior because the base control loop that adjusts the machine's behavior runs more frequently, provided the movement sensors can support the higher frequency. The default base control loop frequency is 10Hz. |
```